src/main/java/bio/terra/workspace/service/buffer/BufferService.java

```diff
@@ -104,4 +111,24 @@ public ResourceInfo handoutResource(HandoutRequestBody requestBody) {
       }
     }
   }
+
+  public SystemStatusSystems status() {
+    UnauthenticatedApi unauthenticatedApi =
+        new UnauthenticatedApi(
+            getApiClient(null).setBasePath(bufferServiceConfiguration.getInstanceUrl()));
+    try {
+      SystemStatus status = unauthenticatedApi.serviceStatus();
+      Map<String, bio.terra.buffer.model.SystemStatusSystems> subsystemStatusMap =
+          status.getSystems();
+      List<String> subsystemStatusMessages =
+          subsystemStatusMap.entrySet().stream()
+              .map(
+                  (entry) ->
+                      entry.getKey() + ": " + StringUtils.join(entry.getValue().getMessages()))
```

Comment:
Do you not want a delimiter here?

Reply:
I copied this from the SamService for consistency...